.github/codecov.yml

```diff
@@ -4,9 +4,9 @@
 codecov:
   notify:
     wait_for_ci: yes
-    after_n_builds: 4 # (unit_tests, bpf2c) * (Debug, Release)
+    after_n_builds: 28 # (unit_tests, bpf2c) * (Debug, Release)
 comment:
-    after_n_builds: 4 # (unit_tests, bpf2c) * (Debug, Release)
+    after_n_builds: 28 # (unit_tests, bpf2c) * (Debug, Release)
```

Comment:
Is it possible to add a comment how we got the number 28. Also, will this number need to be changed, if steps are added to / removed from `cicd.yml`?

Reply:
updated with a comment